go.mod

```diff
@@ -3,112 +3,75 @@ module github.com/influxdata/kapacitor
-	github.com/google/go-cmp v0.5.4
-	github.com/google/uuid v1.0.0
-	github.com/googleapis/gax-go v0.0.0-20170321005343-9af46dd5a171 // indirect
+	github.com/google/go-cmp v0.5.5
+	github.com/google/gofuzz v1.2.0 // indirect
+	github.com/google/uuid v1.1.2
+	github.com/gophercloud/gophercloud v0.17.0 // indirect
 	github.com/gorhill/cronexpr v0.0.0-20180427100037-88b0669f7d75
-	github.com/hashicorp/consul v0.8.1 // indirect
+	github.com/hashicorp/go-hclog v0.14.1 // indirect
+	github.com/hashicorp/go-immutable-radix v1.3.0 // indirect
 	github.com/hashicorp/go-msgpack v1.1.5 // indirect
-	github.com/hashicorp/go-multierror v1.0.0 // indirect
-	github.com/hashicorp/go-rootcerts v0.0.0-20160503143440-6bb64b370b90 // indirect
-	github.com/hashicorp/go-sockaddr v1.0.0 // indirect
 	github.com/hashicorp/go-uuid v1.0.2 // indirect
 	github.com/hashicorp/golang-lru v0.5.4 // indirect
-	github.com/hashicorp/hcl v1.0.0 // indirect
-	github.com/hashicorp/memberlist v0.1.0 // indirect
-	github.com/hashicorp/raft v1.2.0 // indirect
-	github.com/hashicorp/serf v0.8.1 // indirect
-	github.com/influxdata/influxdb v1.1.4
+	github.com/imdario/mergo v0.3.6 // indirect
+	github.com/influxdata/influxdb v1.8.4
 	github.com/influxdata/usage-client v0.0.0-20160829180054-6d3895376368
 	github.com/influxdata/wlog v0.0.0-20160411224016-7c63b0a71ef8
-	github.com/jmespath/go-jmespath v0.0.0-20151117175822-3433f3ea46d9 // indirect
-	github.com/jonboulle/clockwork v0.1.0 // indirect
-	github.com/juju/ratelimit v0.0.0-20170314011755-acf38b000a03 // indirect
 	github.com/jwilder/encoding v0.0.0-20170811194829-b4e1701a28ef // indirect
 	github.com/k-sone/snmpgo v3.2.0+incompatible
 	github.com/kimor79/gollectd v1.0.0 // indirect
-	github.com/kylelemons/godebug v1.1.0 // indirect
 	github.com/mailru/easyjson v0.7.7
-	github.com/mattn/go-runewidth v0.0.2 // indirect
-	github.com/miekg/dns v0.0.0-20170412184748-6ebcb714d369 // indirect
-	github.com/mitchellh/copystructure v0.0.0-20170116004449-f81071c9d77b
-	github.com/mitchellh/go-homedir v0.0.0-20161203194507-b8bc1bf76747 // indirect
-	github.com/mitchellh/mapstructure v0.0.0-20161204053518-5a0325d7fafa
-	github.com/mitchellh/reflectwalk v0.0.0-20170110165207-417edcfd99a4
+	github.com/mattn/go-runewidth v0.0.3 // indirect
+	github.com/mitchellh/copystructure v1.0.0
+	github.com/mitchellh/go-testing-interface v1.14.0 // indirect
+	github.com/mitchellh/mapstructure v1.4.1
+	github.com/mitchellh/reflectwalk v1.0.1
 	github.com/onsi/ginkgo v1.14.2 // indirect
 	github.com/onsi/gomega v1.10.3 // indirect
-	github.com/pborman/uuid v0.0.0-20160209185913-a97ce2ca70fa // indirect
-	github.com/pkg/errors v0.8.1
-	github.com/prometheus/common v0.9.1
-	github.com/prometheus/prometheus v0.0.0-00010101000000-000000000000
+	github.com/pierrec/lz4 v2.5.2+incompatible // indirect
+	github.com/pkg/errors v0.9.1
+	github.com/prometheus/common v0.20.0
+	github.com/prometheus/prometheus v1.8.2-0.20210331101223-3cafc58827d1
```

Comment:
This is actually tag v2.26.0 of prometheus, but https://github.com/prometheus/prometheus/issues/6048 prevents us from using the semver name, we have to specify the commit instead.